### ISSUE TITLE
bug: ElasticMQ is always downloaded, because cache in Dockerfile uses incorrect path

### ISSUE THREAD

**Issue by wimax-grapl** [localstack/localstack#5821]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Upon startup, when elasticmq is enabled, it always downloads the jar instead of using the one downloaded in the Dockerfile.

### Expected Behavior

Use the elasticmq-server.jar downloaded in the Dockerfile.

### How are you starting LocalStack?

With a `docker run` command

### Steps To Reproduce

Long story short:

Upon startup, the following variables have these values:
INSTALL_PATH_ELASTICMQ_JAR is `/opt/code/localstack/localstack/infra/elasticmq/elasticmq-server.jar`
INSTALL_DIR_ELASTICMQ is `/opt/code/localstack/localstack/infra/elasticmq`

We do a download-and-install in installs.py:
```
if not os.path.exists(INSTALL_PATH_ELASTICMQ_JAR):
```

---
Here's the issue - the JAR we download in the Dockerfile goes to the wrong place.
https://github.com/localstack/localstack/blob/ca93a82167092521b42db0cdbfaff3b06d98c83a/Dockerfile#L137
`curl -L -o /opt/code/localstack/localstack/infra/elasticmq-server.jar`

It's missing an `/elasticmq`. 

### Environment

```markdown
- LocalStack: 0.14.2
```

**Comment by wimax-grapl** [comment]
In general, it'd be nice to have some sort of assertion of "I wouldn't expect to be downloading this file" for certain builds. Otherwise failures like this, which slow down startup, are pretty invisible and hard to catch

**Comment by alexrashed** [comment]
Fixed with #5821. Thanks for the contribution!
